### PR TITLE
name case, metadata bullets

### PR DESCRIPTION
**Pull request by davaya**
Initial Cleanup
- Update filename case for consistency: types are capitalized, properties are lower case
- Annotation subject is 1..1
- BOM and SBOM classes are all-caps
- Add missing bullets to metadata
- externalID should be externalId

### CODE REVIEW

**Comment by zvr** [comment]
Thanks for this, David.

-  I thought we'd have all the vocabularies with initial lowercase. (Classes are initial uppercase, properties are initial lowercase as you say)
- I wrote BOM etc. with initial uppercase only on purpose. Why should we have them all caps?
- The rest of the changes are completely correct.

**Comment by davaya** [comment]
If we change BOM and SBOM on the diagram then Bom would be fine.  As long as the model and diagram agree.

Vocabularies are types - properties have vocabulary values.

File: model/Software/Classes/Sbom.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 SPDX-License-Identifier: Community-Spec-1.0
 
-# File
+# SBOM
```
Comment (maxhbr) [inline model/Software/Classes/Sbom.md:3]: I thought we decided for `Sbom` as a class name

**Comment by zvr** [comment]
I think I have incorporated all of these in #60.

File: model/Software/Classes/Sbom.md (hunk 1 above)
Comment (maxhbr) [reply model/Software/Classes/Sbom.md:3]: Ah, the heading does not match the `name` of the class? Feels inconsistent and prone to introduce issues

**Pull request closed without merging**